test/e2e/config/crossbrowser.conf.js

```diff
@@ -30,7 +30,7 @@ const config = {
   },
 
 
-  sauceProxy: 'http://proxyout.reform.hmcts.net:8080',  // Proxy for the REST API
+ // sauceProxy: 'http://proxyout.reform.hmcts.net:8080',  // Proxy for the REST API
```

Comment:
If this isn't needed can it be removed?

Reply:
yes but that might be needed in future if proxy is back if not we remove it in next PRs